handlers/handlers.go

```diff
@@ -49,8 +49,8 @@ func setAuthTokenIfRequired(req *http.Request) ([]dataset.Config, []filter.Confi
 	var filterConfig []filter.Config
 	if len(req.Header.Get("X-Florence-Token")) > 0 {
 		cfg := config.Get()
-		datasetConfig = append(datasetConfig, dataset.Config{InternalToken: cfg.DatasetAPIAuthToken})
-		filterConfig = append(filterConfig, filter.Config{InternalToken: cfg.FilterAPIAuthToken})
+		datasetConfig = append(datasetConfig, dataset.Config{InternalToken: cfg.DatasetAPIAuthToken, FlorenceToken: req.Header.Get("X-Florence-Token")})
```

Comment:
Would it be easier to read if `florenceToken := req.Header.Get("X-Florence-Token")` and then use `florenceToken` on line 50, 52 and 53?